scripts/bootstrap.js

```diff
@@ -36,7 +36,8 @@ function authorize() {
         });
 }
 
-function getOrCreateCollection(collection, token) {
+function getOrCreateCollection(collection, token, attributes) {
+    attributes = attributes || {};
```

Comment:
Sadly, although babel supports these features, I don't think the node.js runtime does:
https://nodejs.org/en/docs/es6/